src/khoj/processor/tools/run_code.py

```diff
@@ -57,15 +57,26 @@ async def run_code(
     except Exception as e:
         raise ValueError(f"Failed to generate code for {query} with error: {e}")
 
+    # Prepare Input Data
+    input_data = []
+    user_input_files: List[FileObject] = []
+    for input_file in input_files:
+        user_input_files += await FileObjectAdapters.async_get_file_objects_by_name(user, input_file)
+    for f in user_input_files:
+        input_data.append(
+            {
+                "filename": os.path.basename(f.file_name),
```

Comment:
Where is this input_data format specification defined? Might be helpful just to include a link to the doc here in case someone tries changing it in the future.

Reply:
Added reference to that in execute_sandbox function below